pkg/build/builder/docker.go

```diff
@@ -267,6 +273,25 @@ func appendEnvVars(fileData string, envVars map[string]string) string {
 	return fileData
 }
 
+//appendBuildLabels appends build labels to a string containing a 
+// valid Dockerfile
+func appendBuildLabels(fileData string, envVars map[string]string) string {
```

Comment:
Instead of creating exact duplicate of `appendEnvVars` rewrite (and rename accordingly) that method to have additional parameter where you'd pass either `ENV` or `LABEL`.